main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
My impression is that if users want to implement something like style, then they wouldn't use the short syntax and would use the full access property definition thingy. =P They would return a special object from the getter that would go back and modify the attribute on the object. Does that cover your use cases?